### PR TITLE
Fix clang-18 build

### PR DESCRIPTION
**Pull request by exdal**
Simply fixes this build error:
```
slang/source/core/slang-performance-profiler.cpp:42:59: error: use of overloaded operator '<<' is ambiguous (with operand types 'StringBuilder' and 'rep' (aka 'long long'))
```

### CODE REVIEW

**Review by jkwak-work — COMMENTED** [review]
I am not sure why this fixes a warning.
`int` is same as `Int32` isn't it?
I don't see anything is different before and after.

Can you share the warning message you were dealing with?

*EDIT: My bad. I found the warning message from the description.
It looks like the change for "int -> Int32" isn't needed, is it?

File: source/core/slang-performance-profiler.cpp [hunk 1]
```diff
@@ -39,7 +39,7 @@ namespace Slang
             for (auto func : data)
             {
                 out << func.key << ": \t";
-                out << func.value.invocationCount << "\t" << func.value.duration.count()/1000000 << "\n";
+                out << static_cast<Int32>(func.value.invocationCount) << "\t" << static_cast<double>(func.value.duration.count()/1000000) << "\n";
```
Comment (jkwak-work) [inline source/core/slang-performance-profiler.cpp:42]: Shouldn't this be more like
```
static_cast<double>(func.value.duration.count())/1000000
```
I think the division should happen after the type is changed to double; not before.

Comment (exdal) [reply source/core/slang-performance-profiler.cpp:42]: You are right, I've completely misread the entire message, further testing with godbolt (https://godbolt.org/z/zYG75axMq), I will be pushing the latest commit.

**Review by jkwak-work — APPROVED** [review]
Looks good to me.
Thank you for adjusting the change quickly.

**Pull request merged**